src/zenml/integrations/skypilot/orchestrators/skypilot_base_vm_orchestrator.py

```diff
@@ -133,8 +133,8 @@ def cloud(self) -> sky.clouds.Cloud:
     def setup_credentials(self) -> None:
         """Set up credentials for the orchestrator."""
         connector = self.get_connector()
-        assert connector is not None
-        connector.configure_local_client()
+        #$assert connector is not None
```

Comment:
Whats this?

Reply:
Not sure why we got rid of this, im sure we need it

Reply:
It was just testing phase that i couldn't do without removing that, it will be back